src/functional_test_runner/lib/proc_runner/index.js

```diff
@@ -0,0 +1 @@
+export { withProcRunner } from './with_proc_runner';
```

Comment:
Maybe move `src/functional_test_runner/lib/proc_runner` to `packages/kbn-proc-runner`? That way we can have a `@kbn/proc-runner` package that Kibana, x-pack and other Kibana plugins can use. (There's a little bit of info in the readme in https://github.com/elastic/kibana/tree/master/packages too, btw)

This code relies on `tooling_log` though, so maybe we want to extract that into a separate package too? (If we did that, we could use that log for `@kbn/pm` at some point too)

/cc @spalger